spring-data-neo4j-rx/src/test/java/org/neo4j/springframework/data/integration/reactive/ReactiveRepositoryIT.java

```diff
@@ -437,6 +439,94 @@ void loadEntityWithRelationshipWithProperties() {
 			.verifyComplete();
 
 	}
+	@Test
+	void persistEntityWithRelationshipWithProperties() {
+		// given
+
+		long personId;
+
+		try (Session session = driver.session()) {
```

Comment:
Any reasons why the nodes get created via Cypher upfront? Why not directly persist a new entity?

Reply:
see above